api/src/Controller/StatsController.php

```diff
@@ -31,4 +33,13 @@ public function getMetric(Request $request)
 
         return $query->execute($params);
     }
+
+    /**
+     * @Route("stats/activeLays", methods={"GET"})
```

Comment:
Would be temped to break this up into a more RESTful set of resources for future reuse. Something like stats/deputies/lay/active

Reply:
Nice - will update 👍